honeybee_energy/load/infiltration.py

```diff
@@ -57,9 +57,10 @@ class Infiltration(_LoadBase):
         * constant_coefficient
         * temperature_coefficient
         * velocity_coefficient
+        * user_data
     """
     __slots__ = ('_flow_per_exterior_area', '_schedule', '_constant_coefficient',
-                 '_temperature_coefficient', '_velocity_coefficient')
+                 '_temperature_coefficient', '_velocity_coefficient', '_user_data')
```

Comment:
Remove from `__slots__` since it's already in the parent class.

Reply:
commit: loads cleaned up